.storybook/gu-preview/GuPreview.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-fallthrough */
```

Comment:
I'm interested in why we need this. If we do need it is it possibly to scope it more tightly? (using the `eslint-disable-next-line` version perhaps?)

Reply:
I'll look at how removing it effects things - it came from the Viewport addon from which I re-used some boiletplate.